File: PROGRAMMERS/lv.1/나머지가1이되는수찾기.py
n = 10
# n = 12


def solution(n):
    answer = 0
    min = 1000000
    for x in range(1, n):
        tmp = n % x
        if tmp == 1 and x < min:
            min = x
    answer = min
    return answer

print(solution(n))

File: PROGRAMMERS/JAVA/src/Lv_1/나머지가1이되는수찾기.java
package Lv_1;

public class 나머지가1이되는수찾기 {
    public int solution(int n) {
        int answer = 0;
        int tmp = 0;
        while (true) {
            if (n % tmp == 1) {
                break;
            }
            tmp++;
        }
        answer = tmp;
        return answer;
    }
}
